creator-node/src/routes/prometheusMetricsExporter.js

```diff
@@ -0,0 +1,33 @@
+const Prometheus = require('prom-client')
+
+const {
+  handleResponse,
+  sendResponse,
+  errorResponseServerError
+} = require('../apiHelpers')
+const PrometheusMetric = require('../services/PrometheusMetric')
+
+const metricPrefix = 'audius_cn_'
+
+// collect default metrics recommended by Prometheus:
+// event loop lag, active handles, GC, and Node.js version
+Prometheus.collectDefaultMetrics({ prefix: metricPrefix })
+
+module.exports = function (app) {
+  app.get(
+    '/prometheus_metrics',
```

Comment:
@SidSethi could you look at the new layout and confirm that is clearer?

I couldn't use `successResponse` since that only returns json objects, not raw text.

re: rate limiting, that's something that we discussed looking into if it became a problem by using a "secret api key" specifically for the `/prometheus_metrics` endpoint.